libs/designsystem/src/lib/components/range/range.component.spec.ts

```diff
@@ -0,0 +1,236 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { IonicModule, IonRange } from '@ionic/angular';
+import {
+  createComponentFactory,
+  createHostFactory,
+  Spectator,
+  SpectatorHost,
+} from '@ngneat/spectator';
+
+import { DesignTokenHelper } from '../../helpers';
+import { TestHelper } from '../../testing/test-helper';
+import { WindowRef } from '../../types';
+
+import { RangeComponent } from './range.component';
+
+const size = DesignTokenHelper.size;
+
+const createComponent = createComponentFactory({
+  component: RangeComponent,
+  declarations: [RangeComponent, IonRange],
+});
+const createHost = createHostFactory({
+  component: RangeComponent,
+  declarations: [RangeComponent, IonRange],
+  imports: [IonicModule.forRoot({ mode: 'ios', _testing: true })],
+  providers: [
+    {
+      provide: WindowRef,
+      useValue: window,
+    },
+  ],
+});
+
+describe('RangeComponent', () => {
+  let component: RangeComponent;
+  let fixture: ComponentFixture<RangeComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [RangeComponent, IonRange],
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(RangeComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should be created', () => {
+    expect(component).toBeTruthy();
+  });
+});
+
+describe('RangeComponent default properties', () => {
+  let spectator: Spectator<RangeComponent>;
+
+  beforeEach(() => {
+    spectator = createHost(
+      `
+        <kirby-range minLabel="Min" maxLabel="Max">
```

Comment:
Looks like it should be refactored, to reflect the current usage of kirby-range?

Reply:
Wilco